rust/processor/src/db/common/models/fungible_asset_models/raw_v2_fungible_asset_activities.rs

```diff
@@ -37,48 +34,27 @@
 pub type CurrentCoinBalancePK = (OwnerAddress, CoinType);
 pub type EventToCoinType = AHashMap<EventGuidResource, CoinType>;
 
-/// TODO: This is just a copy of v2_fungible_asset_activities.rs. We should unify the 2 implementations
-/// and have parquet as an output.
-#[derive(
-    Allocative, Clone, Debug, Default, Deserialize, FieldCount, ParquetRecordWriter, Serialize,
-)]
-pub struct FungibleAssetActivity {
-    pub txn_version: i64,
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct RawFungibleAssetActivity {
+    pub transaction_version: i64,
     pub event_index: i64,
     pub owner_address: Option<String>,
     pub storage_id: String,
     pub asset_type: Option<String>,
     pub is_frozen: Option<bool>,
-    pub amount: Option<String>, // it is a string representation of the u128
-    pub event_type: String,
+    pub amount: Option<BigDecimal>,
+    pub type_: String,
```

Comment:
why rename?

Reply:
I think it's because I copied the postgres model lol. will revert it